src/config/fenix.js

```diff
@@ -0,0 +1,126 @@
+import { transformAPIResponse } from '../utils/transform-data';
```

Comment:
note: this config is not functional until (1) we have the `product` field in the store set on pageload & (2) we are using the keyed config file in `products.js`. I filled this out in anticipation of other blocking work getting finished.